pkg/event/endpoint.go

```diff
@@ -20,7 +20,7 @@ func MakeEventEndpoint(c MuxComponent) cs.Endpoint {
 		case Request:
 			return nil, c.Event(ctx, r.Type, r.Object)
 		default:
-			return nil, fmt.Errorf("request has wrong type: %T", req)
+			return nil, fmt.Errorf("wrongTypeRequest.%T", req)
```

Comment:
I think we should create a specific kind of error. MAybe we could return a JSON strucutre for such error with dynamic values (%T).

Reply:
I think that both JSON and plain text responses are possible but we should decide if we always use JSON or plain text.

Reply:
You are right Francis, it should stay coherent. So ok for me as it is.

Reply:
OK, so no change needed here as agreed with both of you.